Vagrantfile

```diff
@@ -0,0 +1,38 @@
+Vagrant.configure("2") do |config|
+  config.vm.box = "hashicorp/precise64"
+  config.vm.provider "hyperv"
```

Comment:
Do you find you need this line?

https://www.vagrantup.com/docs/providers/configuration

`config.vm.provider` is to specify optional provider-specific configuration by passing in a `do` block. If someone tries to run `vagrant up` without using hyper-v it will silently ignore the section. Without the `do`, I don't think this line does anything at all.

But then in order for it to work with other providers (e.g. VirtualBox) you should include the network config to forward from the host to port 5000 of the VM, even though it is unused by Hyper-V.

Reply:
See what you mean. I removed the line. Added the network line.